service-api/module/Application/src/Fixtures/SsmHandler.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Fixtures;
+
+use Aws\Ssm\SsmClient;
+
+class SsmHandler
+{
+    public function __construct(private readonly SsmClient $ssmClient)
+    {
+    }
+
+    public function getParameter(string $paramName): array
+    {
+        $parameter = $this->ssmClient->getParameter([
+            'Name' => $paramName
+        ])->toArray();
+
+        if (is_string($parameter['Parameter']['Value'])) {
```

Comment:
`$parameter['Parameter']['Value']` is always a string, even when it's not JSON, so this will cause an exception if you used it on a non-JSON parameter.

I'd suggest renaming the method to `getJsonParameter` to be clear about what it's doing.